seed_data/lib.py

```diff
@@ -284,27 +266,45 @@ def ensure_cached_data_freshness(user):
     UserCacheRefreshTime.objects.update_or_create(user=user, defaults=updated_values)
 
 
-def clear_edx_data(user, course=None, course_run=None, models=None):
-    """Clears all of a User's cached edX data for a Course/CourseRun, or just the data for specific cached models"""
+def clear_course_payment_data(user, course=None, course_run=None):
+    """
+    Clears all course payment data (Order/Line) for a given User and an associated Course or CourseRun
+    """
+    program = None
+    course_run_params = {}
+    if course:
+        program = course.program
+        course_run_params['course'] = course
+    elif course_run:
+        program = course_run.course.program
+        course_run_params['id'] = course_run.id
+    if program.financial_aid_availability:
```

Comment:
Could you mention that in the function docstring? In my opinion, it's sufficiently unusual that it should be called out.